docs/basic/repository_setup.md

```diff
@@ -0,0 +1,23 @@
+# Repository setup
+
+## Git integration
+
+To be able to write tests for data factory, you need to have the pipeline and activity definitions available. The recommended way to do this is to sync the Data Factory instance to a git repository, so that you can create a `tests` folder in the same repository and write tests for your data factory. The git integration process can be found here:
+
+1. [Fabric - Git integration process](https://learn.microsoft.com/en-us/fabric/cicd/git-integration/git-integration-process)
+2. [Azure Data Factory - Git integration process](https://learn.microsoft.com/azure/data-factory/source-control)
+
+### Alternative for Azure Data Factory
+
+If you want to download a single JSON file for testing purposes, you can do so by following these steps:
+
+1. Open your Data Factory instance, and open the pipeline you want to test.
+2. Click on the action ellipses
+3. Click "Download support files"
+4. Extract the zip file containing the pipeline definition in a folder of your choice.
+
+> Remember the location of this folder, as you will need it to initialize the framework.
+
+![image](../images/download_support_files.png)
```

Comment:
Either add a descriptive alternative text (compared to [image]) or remove the screenshot entirely. Reasons:
1. screenshots can be outdated and confuse users
2. are not supported for screenreaders (e.g., accessbility)
3. the description above is more than concise enough IMHO!

Reply:
Added description